flask_redis_sentinel.py

```diff
@@ -34,6 +35,7 @@ def __init__(self, url, client_class, client_options, sentinel_class, sentinel_o
         self.sentinel_class = sentinel_class
         self.sentinel_options = sentinel_options
         self.local = Local()
+        self._connections = queue.Queue()
```

Comment:
Hmm, `_connections` is a little bit misleading since technically it contains the sentinel and StrictRedis clients (which have connection pools that contain the connections). The term is used for the cached clients though (`self.local.master_connections` and `self.local.slave_connections`), so I'm not sold either way.